src/Facility.Definition/Fsd/FsdParser.cs

```diff
@@ -49,8 +65,9 @@ public ServiceInfo ParseDefinition(NamedText source)
 
 							var position = new NamedTextPosition(source.Name, headingLineNumber, 1);
 							if (remarksSections.ContainsKey(name))
-								throw new ServiceDefinitionException("Duplicate remarks heading: " + name, position);
-							remarksSections.Add(name, new FsdRemarksSection(name, lines, position));
+								errors.Add(new ServiceDefinitionError("Duplicate remarks heading: " + name, position));
+							else
+								remarksSections[name] = new FsdRemarksSection(name, lines, position);
```

Comment:
Why change from `Add`?

Reply:
I had made it last-one-wins, then changed it back (by adding the else) and forgot to change this.